site/_data/v1-4-toc.yml

```diff
@@ -11,7 +11,13 @@ toc:
     subfolderitems:
       - page: Basic Install
         url: /basic-install
-      - page: Customize Installation
+      :x
```

Comment:
undo!

Reply:
Yep! vim got in a weird state and dumped my commands into the editor.